_posts/2018-02-27-what-should-go-stdlib.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "What Should Go Into the C++ Standard Library"
+layout: blog
+sidenav: side-nav-blog.html
+published: false
+permalink: blog/20180227-what-should-go-stdlib
+type: markdown
+---
+
+By [Titus Winters](mailto:titus@google.com), @TitusWinters
+
+<i><b>About the Author - </b> Titus holds a PhD in Computer Science Education
+from UC Riverside. He created most of Google’s internal C++ coursework, and
+plays an active role in the C++ mentoring program that serves several thousand
+Google C++ engineers.  He has been deeply involved in C++ library infrastructure
+since 2011, through the Google C++ Library Team and now Abseil.  He chairs the
+Library Evolution Working Group (LEWG) - the sub-committee of the C++ Standard
+that focuses on the design of the C++ standard library.</i>
+
+A few interesting pieces on the direction and design for C++ have circulated
+recently.  First off, I strongly recommend everyone read through [Direction for
+ISO C++](wg21.link/P0939r0) ; it provides a lot of excellent detail and
```

Comment:
Fix link. (Needs http:// or https://)

Also, I always start links on a new line if they wrap:

[Direction for ISO C++](https://wg21.link/P0939r0)

Reply:
Done